### DISCUSSION TITLE
Broadcast to WLED

### DISCUSSION THREAD

**Discussion by brakthehun** [arkypita/LaserGRBL#1898] [Ideas]
I hate the sounds but I'm a fan of light indicators.
WLED accepts many forms of communications - https://kno.wled.ge/interfaces/json-api/
I use LEDFX to control them with music and you just give it he IP of the WLED and the pixel count - https://github.com/LedFx/LedFx

I would like to see a color / animation for idle / alarm / running / complete

**Comment by arkypita** [comment]
for feature request please use "discussion" since this is not a bug or an issue

**Reply by SBajonczak** [reply]
@brakthehun  when you use homeassistant too, you can create an automation and using the mqtt integration for that: https://github.com/arkypita/LaserGRBL/pull/2553
